Support/Python/tbdata/brush_lookup.py

```diff
@@ -11,51 +11,67 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# mypy: no-strict-optional
 
 import os
 import re
+from typing import cast, Dict, Iterator, List, Tuple, NewType
 from collections import defaultdict
 
+Guid = NewType('Guid', str)
+
 
 class BrushLookup():
   """Helper for doing name <-> guid conversions for brushes."""
+
+  def iter_standard_brush_guids(self) -> Iterator[Guid]:
+    """Yields all the standard (non-experimental) brush guids"""
+    with open(os.path.join(self.dir, 'Assets/Manifest.asset')) as inf:
+      data = inf.read().replace('\r', '')
+    brush_chunk = re.search(r'Brushes:\n(  -.*\n)*', data).group(0)
+    for match in re.finditer(r'guid: ([0-9a-f]{32})', brush_chunk):
+      yield cast(Guid, match.group(1))
+
   @staticmethod
-  def iter_brush_guid_and_name(tilt_brush_dir):
+  def iter_brush_guid_and_name(tilt_brush_dir: str) -> Iterator[Tuple[Guid, str]]:
+    """Yields (guid, name, is_experimental) tuples."""
```

Comment:
This docstring looks incorrect - where does is_experimental get returned?

Reply:
Done.
Oops! That got added then removed.